### ISSUE TITLE
T2 - Tests

### ISSUE THREAD

**Issue by MrEarle** [PUC-IIC2283/2020-IIC2283-Tareas#17]
Hola, van a subir tests para probar nuestros algoritmos (como en la tarea 1)?

**Comment by mmunos** [comment]
Sí. En algún momento dentro de esta semana. Voy a avisar cuando estén subidos.
